test/regression/0500/0548.stmt

```diff
@@ -0,0 +1,44 @@
+>>> (indent 2)
+  StatementBuilder assignConst(String variable, [TypeBuilder type]) {
+    return new _AsAssign(
+      this,
+      variable,
+      keyword: _AsAssignType.asConst,
+      type: type,
+    )
+        .asStatement();
+  }
+<<<
+  StatementBuilder assignConst(String variable, [TypeBuilder type]) {
+    return new _AsAssign(
+      this,
+      variable,
+      keyword: _AsAssignType.asConst,
+      type: type,
+    ).asStatement();
+  }
```

Comment:
Each pair of snippets in these test cases is before followed by after.

Reply:
What's the behaviour if the method chain need to wrap ?

Reply:
Then it does split before the `.`:

```dart
// If it fits, don't split:
function(
  argument,
).method().method();

// If it splits, split before all of them, even the first:
function(
  argument,
)
    .method()
    .method()
    .method()
    .method()
    .method();
```

Thanks for asking this, it reminded me to add some more test coverage.